docs/network/network_binding_plugins.md

```diff
@@ -175,8 +175,9 @@ A possible alternative to a sidecar, to configure the domain vNIC.
 
 Supported types:
 
-- `tap` (from v1.1.1): The domain configuration is set to use an existing
+- `tap`: The domain configuration is set to use an existing
   tap device. It also supports existing `macvtap` devices.
+- `managedTap`: [From 1.4] Creates and manages a bridge with a tap device, without a DHCP server.
```

Comment:
When no external tap device is provided - it behaves like the core bridge binding, but without the internal DHCP server.
It is used in cases where the network provider can provide DHCP service to the guest.

Commit message: Consider adding a link to the PR which introduced the feature https://github.com/kubevirt/kubevirt/pull/13024.